client/modules/IDE/components/SketchList.jsx

```diff
@@ -145,14 +146,14 @@ class SketchListRowBase extends React.Component {
 
   handleSketchDelete = () => {
     this.closeAll();
-    if (window.confirm(`Are you sure you want to delete "${this.props.sketch.name}"?`)) {
+    if (window.confirm(this.props.t('SketchList.ConfirmDelete', { name_sketch: this.props.sketch.name }))) {
```

Comment:
Why `name_sketch` and not `nameSketch` or `sketchName`?

Reply:
This is also using Common.DeleteConfirmation so is now unified